do/serverless.go

```diff
@@ -678,6 +681,42 @@ func (s *serverlessService) DeleteNamespace(ctx context.Context, name string) er
 	return err
 }
 
+func (s *serverlessService) CleanNamespace() error {
+	// Deletes all triggers
+	ctx := context.TODO()
+	triggers, err := s.ListTriggers(ctx, "")
+
```

Comment:
I understand why errors aren't returned here and the trigger deletion is just skipped.  But, perhaps the reason should be stated in a comment for clarity.